portability-types-common/src/main/java/org/datatransferproject/types/common/models/social/SocialActivityActor.java

```diff
@@ -17,11 +17,14 @@
 package org.datatransferproject.types.common.models.social;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import org.datatransferproject.types.common.ImportableItem;
 
+import javax.annotation.Nonnull;
 import java.util.Objects;
 
-public class SocialActivityActor {
+public class SocialActivityActor implements ImportableItem {
```

Comment:
No import-related usages in this repo, but making it an `ImportableItem` makes sense.